__mocks__/mockCreators/jobs.ts

```diff
@@ -68,9 +87,11 @@ export function createJobsTree(session: imperative.Session, iJob: zowe.IJob, pro
     };
     testJobsTree.mSessionNodes = [];
     testJobsTree.mSessionNodes.push(jobNode);
-    testJobsTree.addFavorite.mockImplementation((newFavorite) => { testJobsTree.mFavorites.push(newFavorite); });
+    testJobsTree.addFavorite.mockImplementation((newFavorite) => {
+        testJobsTree.mFavorites.push(newFavorite);
```

Comment:
Why did you move this to 3 lines? Seems cleaner to have it on 1

Reply:
That is likely webstorm code formatting - similar to other changes in this PR. Once we get prettier we will not have this kind of changes in PRs.